feature-libs/cart/main/root/components/add-to-cart/add-to-cart.module.ts

```diff
@@ -0,0 +1,52 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { ReactiveFormsModule } from '@angular/forms';
+import { RouterModule } from '@angular/router';
+import {
+  CmsConfig,
+  FeaturesConfigModule,
+  I18nModule,
+  provideDefaultConfig,
+  UrlModule,
+} from '@spartacus/core';
+import {
+  IconModule,
+  ItemCounterModule,
+  KeyboardFocusModule,
+  ModalModule,
+  PromotionsModule,
+  SpinnerModule,
+} from '@spartacus/storefront';
+import { AddToCartComponent } from './add-to-cart.component';
+
+@NgModule({
+  imports: [
+    CommonModule,
```

Comment:
except `CommonModule`, `ReactiveFormsModule`, `I18nModule` and `ItemCounterModule`, all other modules are not used, right?

Reply:
Indeed, a cleanup was required here.  AddToCartComponent was previously declared in the same module file as other components and I had carried over all the imports.